db-services/db-palette-service/src/main/java/com/qreal/wmp/db/palette/model/PaletteView.java

```diff
@@ -0,0 +1,44 @@
+package com.qreal.wmp.db.palette.model;
+
+import com.qreal.wmp.thrift.gen.TPaletteView;
+import lombok.Data;
+
+/* View of palette for client side.*/
```

Comment:
I guess there should be a db-services/db-palette-service/README.md providing a top-view explanation of what are the entities here, why we need this service and how it all works. see db-diagram-service's README, for example. also please update the ER data model in UML/Architecture-WMP.vpp